spec/features/flows/sp_authentication_flows_spec.rb

```diff
@@ -10,6 +10,7 @@
 
     it 'prompts the user to log in or sign up' do
       screenshot_and_save_page
+      screenshot_and_open_image
```

Comment:
I'm working with Adam to include screenshots in the output; I left this one here while we work on that output.